wagtail/wagtailadmin/static/wagtailadmin/css/.gitignore

```diff
@@ -0,0 +1,9 @@
+core.css
+layouts/home.css
+layouts/login.css
+layouts/page-editor.css
+layouts/preview.css
+normalize.css
+panels/rich-text.css
+userbar.css
+userbar_embed.css
```

Comment:
I could get on board with that, although `dist` is best matched with a `src` so the structure of `static/wagtailadmin` might be:

```
    css
        vendor
        src (the sass files)
        dist
    images
    js
```